docs/topic-guides/tls.txt

```diff
@@ -0,0 +1,68 @@
+.. _tls:
+
+##########################
+ Transport Layer Security
+##########################
+
+**Transport layer security** (TLS) is a protocol for applying encryption
+to network communications. It prevents the data being transmitted from
+being modified or read while it is in transit. Determined can be set up
+to use TLS for all inbound connections to the master. That means that
```

Comment:
Why only "inbound"? i.e., is this a distinction that will matter to the user? If so maybe we can phrase it differently?